Scripts/hex2rgb.js

```diff
@@ -0,0 +1,25 @@
+/**
+     {
+         "api":1,
+         "name":"HEX2RGB",
+         "description":"Convert color in hexadecimal to RGB.",
+         "author":"Venkat",
+         "icon":"table",
+         "tags":"flip"
+     }
+ **/
+
+function main(input) {
+    R = hexToR(input.text);
+    G = hexToG(input.text);
+    B = hexToB(input.text);
```

Comment:
This solution requires you to exactly select the text that is the #aabbcc format. You can instead use `replace` with a regular expression to iterate through all the matches in the selected text. And also do it with matching groups:

```
return input.text.replace(
  /#([0-9a-f]{2})([0-9a-f]{2})([0-9a-f]{2})/g,
  (_, red, green, blue) => {
    return `${parseInt(red, 16)},${parseInt(green, 16)},${parseInt(blue, 16)}`;
  }
);
```

Reply:
Hey @spoike, that's a very good suggestion. I'm gonna merge this one, would you mind opening a PR adding your upgrades to it? Thanks!